visier/connector/sessions.py

```diff
@@ -218,31 +220,43 @@ def _connect_auth_code(self, auth: OAuth2):
         """Connect to Visier using (three-legged) OAuth2.
         This method will attempt to open a browser for the authentication and consent screens.
         It will also spin up a local web server to receive the OAuth2 authorization code."""
-        url_prefix = auth.host + "/v1/auth/oauth2"
+        def mk_pkce_pair():
```

Comment:
For the life of me I can't exactly figure out what `mk_pkce_pair` stands for - maybe folks who are using this code would know better than me, but maybe can spell out some of it for readability reasons, besides that this is fine.

Reply:
Ok. Will add docstring